includes/core/server/request/class-paginated.php

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * Class file for WCPay\Core\Server\Request\Paginated.
+ *
+ * @package WooCommerce Payments
+ */
+
+namespace WCPay\Core\Server\Request;
+
+use WCPay\Core\Exceptions\Server\Request\Invalid_Request_Parameter_Exception;
+use WCPay\Core\Server\Request;
+use WP_REST_Request;
+
+/**
+ * Request class for paginated requests.
+ */
+class Paginated extends Request {
+
+
+	const DEFAULTS = [
+		'page'      => 0,
```

Comment:
If you take a look at `list_` endpoints in `WC_Payments_API_Client` class, you will find that the default value for page is zero. It felt wired to me too, but I kept it to zero because I didn't want to change previous behavior.

Reply:
It happens to also work with 0, so let's leave it that way then :)